docs/tooling.md

```diff
@@ -0,0 +1,106 @@
+# Developer Tooling Commands
+
+- [Developer Tooling Commands](#developer-tooling-commands)
+  - [Reduce XML command](#reduce-xml-command)
+  - [Analyse XML command](#analyse-xml-command)
+  - [Delete imported page command](#delete-imported-page-command)
+  - [Useful django shell commands](#useful-django-shell-commands)
+    - [Delete all imported images](#delete-all-imported-images)
+    - [Delete all imported documents](#delete-all-imported-documents)
+  - [Errors while running an import](#errors-while-running-an-import)
+  - [Databases](#databases)
+
+Wagtail Wordpress Import makes it easy to work with the raw data by providing a set of command line tools.
+
+## Reduce XML command
+
+This is a command to help you reduce the amount of data that needs to be parsed and imported.
+
+```bash
+python manage.py reduce_xml path/to/your/xmlfile.xml
+```
+
+It will remove all items of `<wp:comment>`. These xml tags represent comments.
+
+The original file is preserved and a new file is created with `-reduced` appended to the file name.
+
+---
+
+## Analyse XML command
+
+This is a command to help you analyse the data you will be importing.
+
+```bash
+python manage.py analyze_html_content path/to/your/xmlfile.xml
+```
+
+It will generate a table output in the console to show inline styles, HTML tags and shortcodes used to format the content.
+
+---
+
+## Delete imported page command
+
+When testing imports you may need to delete the imported pages and run the import again.
+
+```bash
+python manage.py delete_imported_pages [app] [page_model]
+# app and page_model are required arguments
+```
+
+This script will run until all pages have been deleted and displays the  progress in the console.
```

Comment:
**Question**: Does it only delete imported pages, or all objects in the specified model table? Can we clarify this, and make it clear with a warning if it's the worse option?